spikeinterface/toolkit/postprocessing/correlograms.py

```diff
@@ -1,6 +1,71 @@
 import numpy as np
 
 
+try:
+    import numba
+    HAVE_NUMBA = True
+except ModuleNotFoundError as err:
+    HAVE_NUMBA = False
+
+def compute_autocorrelogram_from_spiketrain(spike_train, max_time, bin_size, sampling_f):
```

Comment:
I wanted to do it this way because the cross-correlogram is not symetric, whereas the auto-correlogram is.

This means the second loop is different for both and the auto-correlogram doesn't have to look backwards, which is a nice gain of time.

Also computing an auto-correlogram as a cross-correlogram is a bit tricky as you need to not count the computation with the same spike.

I was thinking of a function `compute_correlogram` with a parameter to ask for cross or auto, and then it redirects to one of the 2 functions?